website/pages/category/restaurant.tsx

```diff
@@ -1,92 +1,94 @@
+                </p>
+                <p className="mb-9 mt-3 text-sm sm:mb-14 sm:text-base">
+                  {item.address}
+                </p>
+                <div className="absolute bottom-0 flex w-full flex-col gap-2">
+                  <div className="flex items-center justify-between font-extrabold">
+                    {item.reviews > 0 ? (
+                      <p className="">
+                        {item.reviews}{" "}
+                        <span className="text-sm font-normal"> Reviews</span>
+                      </p>
+                    ) : (
+                      ""
+                    )}
+                    {item.rating > 0 ? (
+                      <p className="px-4 sm:py-2">⭐ {item.rating}</p>
                     ) : (
-                      <Image
-                        src={item.image as string}
-                        className="h-60 w-full border-b border-black object-cover pb-2 dark:border-white/40 sm:h-[30rem]"
-                        alt="item-image"
-                        height={100}
-                        width={100}
-                      />
+                      ""
                     )}
-                    <p className="absolute -mt-14 w-full bg-black bg-opacity-35 pb-2 pl-5 pt-2 text-xl font-extrabold capitalize text-white dark:border-white sm:text-2xl">
-                      {item.name}
-                    </p>
-                    <p className="mb-9 mt-3 text-sm sm:mb-14 sm:text-base">
-                      {item.address}
-                    </p>
-                    <div className="absolute bottom-0 flex w-full flex-col gap-2">
-                      <div className="flex items-center justify-between font-extrabold">
-                        {item.reviews > 0 ? (
-                          <p className="">
-                            {item.reviews}{" "}
-                            <span className="text-sm font-normal">
-                              {" "}
-                              Reviews
-                            </span>
-                          </p>
-                        ) : (
-                          ""
-                        )}
-                        {item.rating > 0 ? (
-                          <p className="px-4 sm:py-2">⭐ {item.rating}</p>
-                        ) : (
-                          ""
-                        )}
-                      </div>
-                    </div>
-                  </Link>
-                ))}
-              </div>
-            </div>
-          ) : (
-            <div className="mt-20 text-center text-black/50 dark:text-white/70">
-              No restaurant found
-            </div>
-          )}
-        </section>
+                  </div>
+                </div>
+              </Link>
+            ))}
+          </div>
+        </div>
+      ) : isRestaurantsLoading ? (
+        <div className="mt-10 grid h-80 grid-cols-1 gap-x-8 gap-y-10 md:grid-cols-2 md:gap-x-6 lg:gap-x-8 xl:h-60 xl:grid-cols-3">
```

Comment:
How about creating a separate component for animation?

Reply:
but we need to make different different placeholders as per real data view

Reply:
ok